src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/ConsulWatcherStats.java

```diff
@@ -42,6 +44,8 @@ void indexNotChanged() {
         indexNotChangedEvents.incrementAndGet();
     }
 
+    void indexBackwards() { indexBackwardsEvents.incrementAndGet(); }
```

Comment:
Maybe rename to `incrementAndGetIndexBackwards`? :)

Reply:
I simply followed the convention used above (`indexNotChanged`, etc)